### ISSUE TITLE
Scrollbars inside DMs, GCs and channels have wrong styling

### ISSUE THREAD

**Issue by Tecnio** [Vencord/Vesktop#438]
**Describe the bug**

Scrollbars inside channels, DMs and groupchats are out of place and look like default Chromium scrollbars.

**To Reproduce**

1. Go inside any DM, channel or GC.

**Expected behavior**

A normal Discord-like one like in the DMs list.

**Screenshots**
![image](https://github.com/Vencord/Vesktop/assets/50290580/2d1b5d03-c685-4f39-b777-3738f21398d2)
![image](https://github.com/Vencord/Vesktop/assets/50290580/b296ea1e-ee6e-4de3-a059-c156d52c37e3)

**Desktop (please complete the following information):**
 - OS/Distro: Fedora Linux
 - Desktop Environment (linux only): GNOME with Wayland
 - Version: 39

**Command line output**

```
Passing the following arguments to Electron: --ozone-platform-hint=auto
[3:0312/194817.163982:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[3:0312/194817.490761:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[3:0312/194817.490802:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
```

**Comment by Vendicated** [comment]
this is because vesktop now uses chrome 121, which finally added support for some very old scrollbar css properties

this breaks discord's scrollbar styles

you can either fix it yourself via css, or wait for discord to fix it themselves

more context:
https://developer.chrome.com/docs/css-ui/scrollbar-styling
https://syntackle.com/blog/changes-to-scrollbar-styling-in-chrome-121/

**Comment by Vendicated** [comment]
here's a temporary fix: Add this to your QuickCss
```css
* {
    scrollbar-width: unset !important;
    scrollbar-color: unset !important;
}
```

**Comment by Vendicated** [comment]
my css workaround is included natively in v1.5.2 so i'll close this as resolved
